.buildkite/pipeline.yml

```diff
@@ -63,3 +63,22 @@ steps:
     plugins: [*docker_plugin]
     agents:
       queue: "default"
+
+  - group: "Linters"
+    steps:
+      - label: ☢️ Danger - PR Check
```

Comment:
I updated the branch requirements to use this name.

As a follow up, I shall open a new PR implementing the custom GitHub commit status names.